source/exploring_catalog_and_datasets/04_getting_involved/commenting_via_disqus.rst

```diff
@@ -0,0 +1,29 @@
+Commenting via Disqus
+=====================
+
+.. admonition:: Important
+   :class: important
+
+   Since Disqus is an optional feature, it is not activated on every OpenDataSoft portal. The activation of Disqus entirely depends on the portal administrators.
```

Comment:
> The activation of Disqus entirely depends on the portal administrators.

The activation of Disqus _depends entirely_ on the portal administrators. (word reorder)